app.rb

```diff
@@ -120,4 +121,31 @@ def list_rentals
       puts ''
     end
   end
+
+  def save_data
+    save_data_to_json('books.json', @books)
+    save_data_to_json('people.json', @person)
+    save_data_to_json('rentals.json', @rentals)
+  end
+
+  def load_data
+    @books = load_data_from_json('books.json', [])
+    @people = load_data_from_json('people.json', [])
+    @rentals = load_data_from_json('rentals.json', [])
+  end
+
```

Comment:
- [ ] You have not handled the scenario of saving books, properly. Your implementation saved the memory of the book and people, and when I start the app, the JSON files are cleared. I am unable to list the books. Kindly fix that. Your JSON files should keep track of all the data, regardless of whether the app was exited or not. Previously added data should still be present on loading the app.